website/docs/r/log_profile.html.markdown

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
You should add this page URL to [the ToC](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/azurerm.erb#L692).

Reply:
added